2024/json-ld-wg/index.html

```diff
@@ -178,39 +179,62 @@ <h2>Motivation and Background</h2>
+        </p>
+
+
+        <ol>
+          <li>[Placeholder: safe mode]</li>
+          <li>[Placeholder: context file hash, and related stuff]</li>
+        </ol>
+
+        <!-- Note that the JSON-LD APIs are not browser specific; while appropriate for use within browsers, they are not limited to such use, and there is no requirement for browsers to implement them natively. -->
+
+        <p>
+          The development of the 1.2 versions of the JSON-LD specification will follow the principle of
+          backward compatibility. This means that any valid JSON-LD 1.1 document will remain valid for 
+          JSON-LD 1.2, and the RDF graphs generated by the 1.1 and 1.2 versions of the JSON-LD API 
+          specifications, will be <a href="https://www.w3.org/TR/rdf11-concepts/#graph-isomorphism">isomorphic</a>.
+        </p>
+
         <p>
           The Working Group will also develop new Recommendation Track deliverables, based on work incubated by the <a href="https://www.w3.org/community/json-ld/">JSON for Linking Data Community Group</a>,
           specifying the use of JSON-LD algorithms with similar formats (YAML and CBOR).
         </p>
-        <p>The Working Group is expected to coordinate with the <a href='https://www.w3.org/community/json-ld/'>JSON for Linking Data Community Group</a> on consensus-based proposals related to content changes for the JSON-LD Working Group Deliverables. The Chairs of this group may choose to reject proposals that are incompatible with this Charter.</p>
-
-        <section id="section-feature-updates">
-          <h3 id="in-scope">In Scope</h3>
-            <p>In addition to Errata and synchronization with the RDF & SPARQL Working Group, features tracked on the
-              <a href="https://github.com/orgs/w3c/projects/84">Project Management Page</a>
-              are in-scope for updates to the core JSON-LD specifications.</p>
-            <p>The Working Group will also consider <a href="https://www.w3.org/2021/Process-20211102/#allow-new-features">allowing new features</a> in
-              these recommendations, according to <a href="https://www.w3.org/2021/Process-20211102/#revised-rec-features">Section 6.3.11.4 of the W3C
-              process</a>, in order to render future evolutions easier.</p>
-        </section>
+
+        <p>
+          The Working Group is expected to coordinate with the <a href='https://www.w3.org/community/json-ld/'>JSON for Linking Data Community Group</a> on consensus-based proposals related to content changes for the JSON-LD Working Group Deliverables. The Chairs of this group may choose to reject proposals that are incompatible with this Charter.
+        </p>
+
+        <p>
+          Finally, the Working Group will start working on the introduction of the new 
+          <a href="https://www.w3.org/TR/rdf12-concepts">RDF 1.2</a>
+          concepts (primarily triple terms), into the JSON-LD syntax, API, and Framing. The final goal is to 
```

Comment:
```suggestion
          concepts (primarily triple terms and directional language-tagged string), into the JSON-LD syntax, API, and Framing. The final goal is to 
```
Of course JSON-LD 1.1 supports base direction, but does not produce directional language-tagged strings, strictly speaking, so I think it is also worth mentioning.